package.json

```diff
@@ -59,11 +59,13 @@
   "dependencies": {
     "broccoli-funnel": "^1.2.0",
     "broccoli-merge-trees": "^2.0.0",
+    "broccoli-stew": "^1.5.0",
     "ember-cli-babel": "^6.6.0",
     "ember-cli-htmlbars": "^2.0.1",
     "ember-cli-sass": "7.1.3",
     "exists-sync": "^0.0.4",
-    "fastboot-transform": "^0.1.0"
+    "fastboot-transform": "^0.1.0",
+    "materialize-css": "^0"
```

Comment:
The idea here is, by default **this** library is cool with any `0.x` version of materialize-css. If consumers choose to pin to an older version, that's fine too.

Reply:
That makes sense. So a downstream project could specify a particular one as long as it is compatible. What happens if they are not (e.g. someone tries to grab 1.x or something with a special build tag)? Assuming they are compatible, does `npm`/`yarn` guarantee that both will use the same one?